test/sort-imports.test.ts

```diff
@@ -623,41 +623,6 @@ describe(ruleName, () => {
       },
     )
 
-    ruleTester.run(`${ruleName}(${type}): doesn't break user comments`, rule, {
-      valid: [
-        {
-          code: dedent`
-            import { b1, b2 } from 'b'
-
-            /**
-             * Comment
-             */
-
```

Comment:
If there is no blank line between the comment and the import, it will consider this comment as a comment to the import and move it along with it when sorting, right?

Reply:
@azat-io Yes, you can see in 307f8596f308215fe2d5ba96338cc8d15b25a8d1, line 2150, an example where `// Not partition comment` is moved around with its import.